### DISCUSSION TITLE
Nuxt version support

### DISCUSSION THREAD

**Discussion by raphaelbernhart** [raphaelbernhart/nuxtjs-cookieconsent#4] [Polls]
I would like to continue the development of the module but atm there are a few open questions for the future of this project.

One of the most important is the support of the nuxt version. This module is currently based on _**Nuxt v3**_.

**Comment by tmlmt** [comment]
I think that the module should support Nuxt 3 as a minimum, Nuxt 3 has just reached stable version, all new Nuxt projects will start this way, and many Nuxt2 projects will migrate. You can imagine a compatibility with Nuxt 2 if ever there is more demand. 

Thanks for starting this up 😊 I am new to Nuxt so the subtleties of creating Plugins or Modules a bit above my current capabilities but I'll support if I can. I will stay not-GDPR compliant in the meantime 😬 Looking forward to using this in my Nuxt project, so I hope to see a deployable version finished soon.
